tools/cloud-build/daily-tests/builds/hpc-slurm-chromedesktop.yaml

```diff
@@ -15,7 +15,7 @@
 ---
 tags:
 - m.chrome-remote-desktop
-- m.filestore
+- m.filestore # TODO: mark as removed during pre-deploy
```

Comment:
Here is my concerns

1. Divergence between test and example: You could imagine some class of bug that we will miss due to this change.  CRD + Filestore + Slurm. While it seems unlikely, we should consider the value of any deviation between public example and test. What is the time savings for this test?
2. Loss of overall coverage: Maybe for 1) above we decide that in this case the test speedup is worth the divergence. Probably this would be ok in isolation because we have other Filestore + Slurm tests. Where do we holistically evaluate the matrix (A: nowhere)?

For 1), can you categorize the time savings? If it is meaningful then I would agree to try this out. 2) only becomes a concern if we start to apply this to more tests. Do you have any ideas or thoughts on 2).